zenith/README.md

```diff
@@ -0,0 +1,55 @@
+# Dagger: standardize your dev environments
+
+## What is Dagger?
+
+Dagger defines a new standard for packaging, distributing and running software environments as a containerized DAG. Once "daggerized", environments have the following properties:
+
+- **Portable**: any machine that can run standard containers can run a DAG.
+- **Familiar**: keep using the tools you know, packaged in containers.
+- **Scriptable**: automate custom tasks in any language, using a powerful API and growing list of SDKs
+- **Extensible**: easily incorporate the capabilities of other environments into your own
+- **Scalable**: all operations in the DAG are scheduled concurrently, which allows seamless parallelization across cores and if needed, across machines in a cluster (*coming soon*)
+- **Fast**: all operations in the DAG are continuously cached, making all computation magically incremental. As a result, daggerizing an environment typically makes builds and other cpu-intensive tasks much faster out of the box.
+
+## What kind of environments?
+
+In theory, any software environment can be daggerized, as long as its individual components can run in a container. In practice, Dagger is most commonly used for non-production environments, such as:
```

Comment:
> Dagger is most commonly used for non-production environments

I don't think this is accurate. Do you mean "outside the application runtime" instead of "non-prod environments"? In practice, a release process or a CI systems are both production environments where Dagger is used.

Reply:
I think this means that CI is part of the development environment and that it isn't a separate entity that sits in limbo between development and production.

Reply:
I think the disconnect here is that some folks consider CI/CD "production" if it implements the "path to production" whereas this comment is more saying that you don't literally use Dagger to run your actual production environments (like the running webserver etc).

Reply:
Most people use "production environment" in a sense that does not include their CI. So I used the terminology in that way. But we can make it work either way, it's not core to Dagger or Zenith. We just need to make sure we use the words in a way that doesn't confuse our audience.